### DISCUSSION TITLE
[Windows] System Tray icon is missing

### DISCUSSION THREAD

**Discussion by BaituLime** [dynobo/normcap#330] [Q&A]
![image](https://user-images.githubusercontent.com/50917215/208692729-b42ff7ca-d582-454f-b6f9-40fece1f3bb3.png)
I can't see any icon in system tray.

**Comment by dynobo** [comment]
Hi @BaituLime, 

can you please open a CMD terminal and start NormCap via the following command? :
```
`%LOCALAPPDATA%\Programs\dynobo\NormCap\NormCap.exe -- -v debug`
```
After that, please post the output that was printed to the terminal here. This will help to diagnose the problem.

Thanks!
